experiment/project.clj

```diff
@@ -10,7 +10,7 @@
                  [thinktopic/think.gate "0.1.3"]
                  ;;This had better precisely match the version of figwheel that think.gate uses
                  ;;Tried with 1.9.XXX and had odd unexplainable failures.
```

Comment:
I didn't see any instability during my testing - but would benefit from another pair of eyes